pkg/cmd/cli/cmd/newapp.go

```diff
@@ -233,13 +233,14 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
 	}
 
 	mapper, _ := f.Object()
-	var afterFn func(*resource.Info, error)
+	var afterFn configcmd.AfterFunc
 	switch {
 	// only print success if we don't have installables
 	case !result.GeneratedJobs:
 		afterFn = configcmd.NewPrintNameOrErrorAfterIndent(mapper, shortOutput, "created", out, c.Out(), indent)
 	default:
 		afterFn = configcmd.NewPrintErrorAfter(mapper, c.Out())
+		afterFn = configcmd.HaltOnError(afterFn)
```

Comment:
do we really want to halt on error for normal app creation? Until now if you already have an existing resource we say we couldn't create it but still create all the other ones. I guess it's ok if we change the behavior, just want to make sure we're making a conscious decision to do so.